project3/index.html

```diff
@@ -0,0 +1,232 @@
+                            fill="#fff" />
+                        <circle cx="126" cy="12" r="2" fill="#fff" />
+                        <path
+                            d="M106.96 18.28c.29 0 .58-.06.86-.19 1-.47 1.42-1.67.95-2.66-.48-1-1.67-1.43-2.67-.95-1 .47-1.42 1.66-.95 2.66.35.72 1.06 1.14 1.81 1.14zM106.14 9.38c.29.14.59.21.88.21a2.004 2.004 0 00.89-3.8 2.002 2.002 0 00-1.77 3.59z"
+                            fill="#fff" />
+                        <circle cx="116" cy="12" r="2" fill="#fff" />
+                        <circle cx="10" cy="76" r="2" fill="#fff" />
+                        <circle cx="10" cy="38" r="2" fill="#fff" />
+                        <path d="M12 46a2 2 0 00-4 0v22a2 2 0 004 0V46z" fill="#fff" />
+                        <circle cx="118" cy="76" r="2" fill="#fff" />
+                        <circle cx="118" cy="38" r="2" fill="#fff" />
+                        <path d="M120 46a2 2 0 00-4 0v22a2 2 0 004 0V46z" fill="#fff" /></svg>
+                </div>
+                <span> Intelligence</span>
+                <select class="selectors" name="intelligence" id="intelligence">
+                    <option value="within">Within Human Norms</option>
+                    <option value="beyond">Beyond Human Norms</option>
+                </select>
+            </label>
+        </fieldset>
+
+
+        <fieldset>
+            <span>
+                <input id="agree-terms" type="checkbox">
+                <label for="agree-terms" class="confirm-text">I confirm the above information represents my current
+                    details and is
+                    correct to the best of my knowledge</label>
+
+            </span>
+
+            <button type="submit">Register Now</button>
+        </fieldset>
+    </div>
```

Comment:
You can delete the closing `div` and keep the closing `form` after updating line 33.